### PR TITLE
AB Email Notifications - UI Fixes

### PR DESCRIPTION
**Pull request by HasnatL-Adapptlimited**
# Links 
[Base PR](https://github.com/ReliefApplications/ems-frontend/pull/2460)

# QA Tickets

- [ ] Both back and next buttons are disabled in the layout page - Fixed
- [ ] Space not used well on Distribution list page - Fixed

# Description
Fixed Adappt QA points from last push

# Screenshots
### Before:
image
image
image

### After:
image
image
image

### CODE REVIEW

File: libs/shared/src/lib/components/email/components/email-template/email-template.component.html [hunk 1]
```diff
@@ -1,207 +1,113 @@
...
-            </td>
-          </ng-container>
-        </tr>
-      </tbody>
-    </table>
-    <div class="button-wrapper">
-      <button (click)="addSelectedEmails()" *ngIf="selectedItemIndexes.length">
-        Apply
-      </button>
+    <div class="flex flex-row">
```
Comment (AntoineRelief) [inline libs/shared/src/lib/components/email/components/email-template/email-template.component.html:2]: @HasnatL-Adapptlimited 
Which code formatter are you using?

In all front-end PRs, I can see the html files, when edited, always use a different format than the one I push on the main branch

On our side, we use prettier, with the configuration shared in the repo

Comment (AntoineRelief) [reply libs/shared/src/lib/components/email/components/email-template/email-template.component.html:2]: @HasnatL-Adapptlimited 
If you can please check this comment
It's affecting all the PRs, and it's hard sometimes to read the changes due to that

Comment (morgang-adappt) [reply libs/shared/src/lib/components/email/components/email-template/email-template.component.html:2]: Hi @AntoineRelief CC @HasnatL-Adapptlimited 

Apologies for this, I believe some of our developers may be missing some configuration in their editor for linting. Should we be using Prettier as the linter, and do we need any extra configuration for ESLint+Prettier? Have there been any linting issues on the backend?

Comment (AntoineRelief) [reply libs/shared/src/lib/components/email/components/email-template/email-template.component.html:2]: @morgang-adappt @HasnatL-Adapptlimited 

If possible, prefer to use prettier indeed

all configuration for eslint and prettier should be in the repository

**Comment by AntoineRelief** [comment]
@HasnatL-Adapptlimited 

merged with a few changes

**Pull request merged**